src/azure/apim/ApimService.ts

```diff
@@ -306,6 +306,21 @@ export class ApimService {
         return <IMcpServerApiContract[]>(result.parsedBody.value);
     }
 
+    public async createMcpServer(mcpApiName: string, mcpServerPayload: any): Promise<IMcpServerApiContract> {
+        const client: ServiceClient = new ServiceClient(this.credentials, clientOptions);
+        const result: HttpOperationResponse = await client.sendRequest({
+            method: "PUT",
+            url: `${this.baseUrl}/apis/${mcpApiName}?api-version=2024-10-01-preview`,
+            body: mcpServerPayload
+        });
+        
+        if (result.status >= 400) {
+            throw new Error(result.bodyAsText ?? `Failed to create MCP Server. Status code: ${result.status}`);
```

Comment:
What if it returns 500?

Reply:
I didn't distinguish client or server http status code. As long as the request failed, the extension will show related error message to user.